bedrock/firefox/templates/firefox/hello/index-2016.html

```diff
@@ -0,0 +1,134 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "firefox/base-resp.html" %}
+
+{% set_lang_files "firefox/hello-2016" %}
+
+{% block page_title_prefix %}{{ _('Firefox Hello') }} — {% endblock %}
+{% block page_title %}{{ _('Browse the Web with friends') }}{% endblock %}
+{% block page_desc %}{{ _('Use Firefox Hello to browse Web pages with your friends. Make decisions together in real time when planning a trip, shopping and more.') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/hello/hello-share.jpg') }}{% endblock %}
+{% block body_id %}firefox-hello{% endblock %}
+{% block body_class %}mozID{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_hello_2016' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-hello-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% call fxfamilynav('desktop', 'hello') %}
+    <a class="try-hello-button" href="https://support.mozilla.org">{{ _('Try Hello now') }}</a>
+    <a href="{{ url('firefox.new') }}">{{ _('Download now') }}</a>
+  {% endcall %}
+{% endblock %}
+
+{% block content %}
+  <header id="try-hello">
+    <div class="content">
+      <h1>{{ _('Plan. Shop. Decide. Together.') }}</h1>
```

Comment:
Can you define "visually formatted"? We already have [13 locales](https://l10n.mozilla-community.org/~pascalc/langchecker/?locale=all&website=0&file=firefox/hello-2016.lang), and I expect a few more after the weekend.

I honestly don't want to start a brand new page with a l10n tag already in it, but it's also extremely confusing to change a string only 2 days after we started localizing it :-\